README.md

```diff
@@ -36,6 +36,28 @@ You should be able to use the [#test-bot] channel to do your tests or create you
 `npm run lint`
 2. Commit and push your changes, then open up a pull request :rocket:
 
+
+## Troubleshooting
+
+1. if you have this error :
+ 
+ `
+  Error: Cannot find module 'dotenv'
+      at Function.Module._resolveFilename (module.js:455:15)
+      at Function.Module._load (module.js:403:25)
+      at Module.require (module.js:483:17)
+      at require (internal/module.js:20:19)
+      at Object.<anonymous> (/Users/Amine/PersonalProject/dz-community-bot/src/go.js:1:79)
+      at Module._compile (module.js:556:32)
+      at Object.Module._extensions..js (module.js:565:10)
+      at Module.load (module.js:473:32)
+      at tryModuleLoad (module.js:432:12)
+      at Function.Module._load (module.js:424:3)
+  error Command failed with exit code 1. `
+  
+ Make sure you do an ` npm install ` or ` yarn install`. before running `yarn start` or `npm start`
```

Comment:
But, this is already mentioned above, so i guess no need to add this part ! cc @IOAyman